### DISCUSSION TITLE
mean Dice of multi-class segmentation

### DISCUSSION THREAD

**Discussion by YingdiZhang** [Project-MONAI/MONAI#4209] [Q&A]
Thanks for the great work.
I wonder how to calculate the mean dice in a multi-class segmentation task.
It means for example in a 7 class segmentation task, I want to get  one mean dice instead of 7 seperate dice

**Comment by Nic-Ma** [comment]
Hi @YingdiZhang ,

Thanks for your interest here.
MONAI metrics support different reduction methods during aggregation:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/metrics/meandice.py#L38
You can select the expected method.

Thanks.
